### PR TITLE
Healthcare 13148 hide in progress message

### PR DESCRIPTION
**Pull request by mdewey**
## Description
On the Introduction page of the form, the current wording is redundant(see the before picture). Our team doesn't have any wording that makes sense to go there that isn't extra to the experience.  I would like to propose a way to hide that extra message with a boolean toggle. 

I am not a fan of where the boolean lives (`formConfig.saveInProgress.messages.hideInProgressMessage`). I like the pattern of having an option not to have that message. 

## Testing done
- [ ] None yet, but will once the idea gets some traction

## Screenshots

Before
Screen Shot 2020-10-23 at 1 50 36 PM

After
Screen Shot 2020-10-23 at 1 57 19 PM

## Acceptance criteria
- [ ] Unit tests passing
- [ ] Documentation is added. 

## Definition of done
- [ ] [Orginal Ticket for our use case](https://github.com/department-of-veterans-affairs/va.gov-team/issues/13148)

### CODE REVIEW

**Comment by bkjohnson** [comment]
Hey Mark, this pointed out a bug in the forms library that I'm going to try to quickly fix. What _should_ be possible is for the message to be disabled by setting `inProgress` to an empty string. I'm writing up an issue for this now and will get a PR out afterwards.

File: src/applications/healthcare/questionnaire/config/form.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ const formConfig = {
   formId: VA_FORM_IDS.FORM_HC_QSTNR,
   saveInProgress: {
     messages: {
+      hideInProgressMessage: true,
```
Comment (bkjohnson) [inline src/applications/healthcare/questionnaire/config/form.js:33]: This should do the trick now :crossed_fingers: 

```suggestion
      inProgress: '',
```

**Review by bkjohnson — COMMENTED** [review]
If you revert the changes to the forms library and merge in master, you should get the changes from #14820 which should do the trick.

**Comment by mdewey** [comment]
Awesome work @bkjohnson! I have tested locally and it works. I am going to close this PR and add that change into another ticket I am working on that page for simplicity. 

Thanks again!

**Pull request closed without merging**